GetFitLA_v3/app/src/main/res/layout/list_layout.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Always use a Relative Layout in your xmls, a relative layout allows the user to rotate their screen and the layout will auto-adjust to the new orientation.